evalml/pipelines/components/utils.py

```diff
@@ -6,7 +6,9 @@
 
 from evalml.exceptions import MissingComponentError
 from evalml.model_family.utils import handle_model_family
-from evalml.pipelines.components import ComponentBase, Estimator, Transformer
+from evalml.pipelines.components.component_base import ComponentBase
+from evalml.pipelines.components.estimators.estimator import Estimator
+from evalml.pipelines.components.transformers.transformer import Transformer
```

Comment:
Split these to avoid circular imports when we import `make_balanced_dictionary` to our oversampler file